test/Confluent.SchemaRegistry.Serdes.UnitTests/ProtoSerializeDeserialize.cs

```diff
@@ -62,15 +62,20 @@ message ReferencedMessage {
 
             IDictionary<string, string> imports = new Dictionary<string, string>();
             imports["ref.proto"] = import;
+            imports["confluent/meta.proto"] = "doesn't matter, will be overwritten anyway";
 
             var fds = ProtobufUtils.Parse(schema, imports);
-            foreach (var file in fds.Files)
-            {
-                foreach (var messageType in file.MessageTypes)
-                {
-                    Assert.Equal("ReferrerMessage", messageType.Name);
-                }
-            }
+            Assert.Equal(4, fds.Files.Count);
```

Comment:
This updated test now verifies the file descriptor set was actually parsed properly